cmd/glitzz/main.go

```diff
@@ -4,17 +4,42 @@ import (
 	"fmt"
 	"github.com/boreq/guinea"
 	"github.com/lovelaced/glitzz/cmd/glitzz/commands"
+	"github.com/lovelaced/glitzz/logging"
 	"math/rand"
 	"os"
 	"time"
 )
 
 func main() {
 	rand.Seed(time.Now().UnixNano())
-
+	injectGlobalBehaviour(&commands.MainCmd)
 	err := guinea.Run(&commands.MainCmd)
 	if err != nil {
 		fmt.Fprintln(os.Stderr, err)
 		os.Exit(1)
 	}
 }
+
+func injectGlobalBehaviour(cmd *guinea.Command) {
```

Comment:
I am not a big fan but this is a workaround for guinea which is supposed to avoid adding those flags and code to every single command manually.